ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/project/MutableProjectConfig.java

```diff
@@ -124,6 +130,17 @@ public void setSource(SourceStorage sourceStorage) {
         this.sourceStorage = new MutableSourceStorage(sourceStorage);
     }
 
+    public Map<String, String> getOptions() {
+        if (options == null) {
+            options = new HashMap<>();
```

Comment:
`options` is already initialized in constructor.
So this lazy initialization is unnecessary.